.chloggen-aws/sdk-telemetry-user-agent.yaml

```diff
@@ -0,0 +1,21 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
naive q: should we be making a chloggen entry whenever we contribute to our contrib repo?

Reply:
We decided to do it to keep track of our changes, but it isn't enforced well.